src/check.h.in

```diff
@@ -977,6 +1012,11 @@ CK_DLL_EXP void CK_EXPORT srunner_run_all(SRunner * sr,
  * @param sr suite runner where the given suite or test case must be
  * @param sname suite name to run. A NULL means "any suite".
  * @param tcname test case name to run. A NULL means "any test case"
+ * @param include_tags space separate list of tags. Only run test cases
+ * that share one of these tags. A NULL means "any test case".
+ * @param exclude_tags space separate list of tags. Only run test cases
+ * that do not share one of these tags. A NULL means "any test case".
+ * Overrides any include criteria.
```

Comment:
Oh, never mind. I thought the comments were over another function. These are fine.